src/components/NFTPortfolio/MainView.vue

```diff
@@ -323,6 +366,22 @@ export default {
           return 0;
         });
     },
+    creatorFilterLabelText() {
+      const isSelected = this.portfolioCollectedCreatorListWithSorting.filter(
+        creator => creator.isSelected
+      );
+      return isSelected.length
+        ? this.$t('filter_menu_creator_selected', { num: isSelected.length })
+        : this.$t('filter_menu_creator');
+    },
+    nftTypeOptions() {
+      return Object.values(this.portfolioItemsTypeOptions);
```

Comment:
I think this list of options should return translated labels
```suggestion
      return Object.values(this.portfolioItemsTypeOptions).map((value) => ({
        value,
        label: this.getPortfolioTypeFilteringLabel(value),
      });
```